### PR TITLE
migration site updates

### PR DESCRIPTION
**Pull request by mdrillin**
Addressing some issues with website-migration, adding minifier
- added LICENSE.txt
- canonical link now added if present in blog post front matter.  fixed blog posts
- fix blog post author
- adding some files to excludes in the _config.yml
- adding minifier plugin

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -63,6 +64,14 @@ exclude:
   - vendor/cache/
   - vendor/gems/
   - vendor/ruby/
+  - ANTORA.md
+  - CONTRIBUTING.md
+  - LICENSE.txt
+  - playbook.yml
+  - playbook_author.yml
+  - README.md
+  - yarn.lock
```
Comment (gunnarmorling) [inline _config.yml:73]: Do we actually need this file? Are we using Yarn at all even?

File: _posts/2017-02-22-Debezium-at-WePay.adoc [hunk 2]
```diff
@@ -1,5 +1,6 @@
 ---
 layout: post
+canonical_url: https://wecode.wepay.com/posts/streaming-databases-in-realtime-with-mysql-debezium-kafka
```
Comment (gunnarmorling) [inline _posts/2017-02-22-Debezium-at-WePay.adoc:3]: Cool; much nicer than before, putting the canonical link as an attribute right into files themselves 👍

**Review by gunnarmorling — APPROVED** [review]
LGTM, leaving it to @uidoyen and @indraraj to take a look at the rendered site with the minifier. I think with this PR merged we should be good to go for the cut-over tomorrow,  unless you folks identify any blocker issues from reviewing the full diff in PR #589.

File: _config.yml (hunk 1 above)
Comment (mdrillin) [reply _config.yml]: yeah, shoulda removed - I amended the commit

**Comment by indraraj** [comment]
Tested locally rendering minified site.

**Review by gunnarmorling — APPROVED** [review]

**Comment by gunnarmorling** [comment]
Merged, thx!

**Pull request merged**